### PR TITLE
Update templates

### PR DESCRIPTION
**Pull request by jsturek**

### CODE REVIEW

**Comment by jsturek** [comment]
Do we have a way to test .jsp files?

**Comment by ericras** [comment]
> Do we have a way to test .jsp files?

No, idk who specifically is using them and they may very well just rip apart the cod eto fit it into their app anyway

File: wdn/templates_5.0/includes/global/head-2.html [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
 </style>
-<link id="core-css" rel="preload" href="/wdn/templates_5.0/css/core.css?dep=$DEP_VERSION$" as="style" onload="this.onload=null;this.rel='stylesheet';">
+<link id="core-css" rel="preload" href="https://unlcms.unl.edu//wdn/templates_5.0/css/core.css?dep=$DEP_VERSION$" as="style" onload="this.onload=null;this.rel='stylesheet';">
```
Comment (ericras) [inline wdn/templates_5.0/includes/global/head-2.html]: Have some double slashes here and below

Comment (jsturek) [reply wdn/templates_5.0/includes/global/head-2.html]: fixed

**Pull request merged**